generate-db.js

```diff
@@ -21,9 +22,11 @@ if (!TERMUX_ARCH) {
 }
 
 const binPrefix = TERMUX_PREFIX.substring(1) + "/bin/";
+/**@type {unknown}*/
 const repos = JSON.parse(
```

Comment:
This is a JSON object corresponding to https://github.com/termux/termux-packages/blob/master/repo.json

Reply:
Thanks! I had replaced the `any` by `unknown` because there's no absolute guarantee that the input `repo.json` honors the schema. But I guess it's fine to make some assumptions there